translate/storage/placeables/strelem.py

```diff
@@ -881,12 +882,12 @@ def prune(self):
                 # Remove empty strings or StringElem nodes
                 # (but not StringElem sub-class instances, because they
                 # might contain important (non-rendered) data.
-                if (type(elem.sub[i]) in (StringElem, str, unicode) and
-                    len(elem.sub[i]) == 0):
+                if ((type(elem.sub[i]) == StringElem or isinstance(elem.sub[i], six.string_types)) and
+                        len(elem.sub[i]) == 0):
```

Comment:
Please remove the added indent in this last line so it aligns with the parenthesis.